src/chrome/crdpMultiplexing/webSocketToLikeSocketProxy.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import { logger } from 'vscode-debugadapter';
+import * as WebSocket from 'ws';
+import { LikeSocket } from 'noice-json-rpc';
+
+export class WebSocketToLikeSocketProxy {
+    private _server: WebSocket.Server;
+
+    constructor(private _port: number, private _socket: LikeSocket) { }
+
+    public start(): void {
+        this._server = new WebSocket.Server({ port: this._port }, () => {
```

Comment:
We didn't modify the existing logic at all, and the existing logic connects to a chrome-like websocket at a specified address e.g.: ws://localhost:1234/someUriHere so we need to maintain that unless we want to update the other side to support some different method of communication